src/netconfig.rs

```diff
@@ -32,25 +20,34 @@ pub fn read_netconfig(path: impl AsRef<Path>) -> Result<Option<Netconfig>, anyho
 
 fn handle_netconfig_values() -> Result<Option<Netconfig>, anyhow::Error> {
     let mut netconfig = Netconfig::default();
+
     if let Ok(dns_policy) = dotenv::var("NETCONFIG_DNS_POLICY") {
         if dns_policy == "auto" {
             netconfig.dns_policy = vec!["STATIC".to_string(), "*".to_string()];
         } else if !dns_policy.is_empty() {
             if dns_policy.contains(&"STATIC_FALLBACK".to_string()) {
-                anyhow::bail!("NETCONFIG_DNS_POLICY \"STATIC_FALLBACK\" is not supported");
+                netconfig
+                    .warnings
+                    .push("NETCONFIG_DNS_POLICY \"STATIC_FALLBACK\" is not supported".to_string());
+            } else {
+                netconfig.dns_policy = dns_policy.split(' ').map(|s| s.to_string()).collect();
             }
-
-            netconfig.dns_policy = dns_policy.split(' ').map(|s| s.to_string()).collect();
         }
     }
     if let Ok(static_dns_servers) = dotenv::var("NETCONFIG_DNS_STATIC_SERVERS") {
         if !static_dns_servers.is_empty() {
-            netconfig.static_dns_servers = Some(
-                static_dns_servers
-                    .split(' ')
-                    .map(|s| s.to_string())
-                    .collect::<Vec<String>>(),
-            );
+            netconfig.static_dns_servers = static_dns_servers
+                .split_whitespace()
```

Comment:
Mhm a bit of a nitpick but everywhere else we split with `split(' ')` but here it's `split_whitespace()`. But I am fine with leaving it.